### ISSUE TITLE
Why is softmax not applied at training time?

### ISSUE THREAD

**Issue by anuar12** [facebookresearch/SlowFast#20]
I noticed activation (which is softmax) is only applied at test time. Why is that? Usually it is trained with softmax, shouldn't the CrossEntropy loss not work because there would be negative outputs and log is not defined on negative domain?

```
        # Performs fully convolutional inference.
        if not self.training:
            x = self.act(x)
            x = x.mean([1, 2, 3])
```

Also why the `self.projection` said to be fully-convolutional? It is a Fully-Connected layer so in principle it shouldn't be invariant to the input size of the image. I guess you just mean that the weights are initialized similar to Conv layers, right?

**Comment by anuar12** [comment]
Oh right, the `CrossEntropyLoss()` has `LogSoftmax` in it implicitly.
